doc/website/blog/2023-08-13-visualize-with-atlas.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Quickly Generate ERDs from your Ent Schemas (Updated)" 
+author: Rotem Tamir
+authorURL: "https://github.com/rotemtam"
+authorImageURL: "https://s.gravatar.com/avatar/36b3739951a27d2e37251867b7d44b1a?s=80"
+authorTwitter: _rtam
+image: "https://atlasgo.io/uploads/ent/inspect/entviz.png"
+---
+
+### TL;DR
+
+Create a visualization of your Ent schema with one command:
+
+```
+atlas schema inspect --dev-url 'sqlite://demo?mode=memory&_fk=1' -u ent://ent/schema --visualize
+```
+
+![](https://entgo.io/images/assets/erd/edges-quick-summary.png)
+
+
+Hi Everyone!
+
+A few months ago we shared [entviz](/blog/2023/01/26/visualizing-with-entviz), a cool
+tool that enables you to visualize your Ent schemas. Due to its success and popularity
+we decided to integrate it directly in [Atlas](https://atlasgo.io), the migration engine
+that Ent uses. 
+
+Since the release of [v0.13.0](https://atlasgo.io/blog/2023/08/06/atlas-v-0-13) of Atlas,
+you can now visualize your Ent schemas directly from Atlas without needing to install an
+additional tool.
+
+### Private vs. Public Visualizations
+
+Previously you could only share a visualization of your schema to the
+[Atlas Public Playground](https://gh.atlasgo.cloud/explore). While this is convenient
+for sharing your schema with others, it is not acceptable for many teams who maintain
+schemas that themselves are senstive and cannot be shared publicly.
+
+With this new release, you can easily publish your schema direclty to your private
+workspace on [Atlas Cloud](https://atlasgo.cloud). This means that only you and your
+team can access the visualization of your schema.
+
+### Visualizing your Ent Schema with Atlas
+
+To visualize your Ent schema with Atlas, first install its latest version:
+
+```
+curl -sSfL https://atlasgo.io/install.sh | sh
+```
+For other installation options, see the [Atlas installation docs](https://atlasgo.io/getting-started#installation).
+
+Next, run the following command to generate a visualization of your Ent schema:
+
+```
+atlas schema inspect -u ent://ent/schema --dev-url 'sqlite://demo?mode=memory&_fk=1'  --visualize
+```
+
+Let's break this command down:
+* `atlas schema inspect` - this is command can be used to inspect schemas from a variety of sources and output
```

Comment:
```suggestion
* `atlas schema inspect` – this command can be used to inspect schemas from a variety of sources and outputs
```

Reply:
I wasn't sure about "output" vs. "outputs," but I figured I would learn something if you said it was already correct @rotemtam